File: doc/home/eg_gline.m
qfigure('eg_gline', 1.8, 2.);
xx=[-1.5*pi:1.5*pi];
yy=cos(xx);
qmarker o solid 3
qmark(xx, yy);
N=length(xx)-1;

qpen 777
for n=1:N
    qgline({'absdata',xx(n),yy(n), 'retract', 5}, ...
            {'absdata',xx(n+1),yy(n+1), 'retract', 5});
end

qshrink
qsave pdf
qsave png 120

File: doc/home/eg_gline.py
#!/usr/bin/python3

import numpy as np
import qplot as qp

qp.figure('eg_gline', 1.8, 2.)
xx = qp.arange(-1.5*np.pi, 1.5*np.pi, 1)
yy = np.cos(xx)
qp.marker('o', 3, 'solid')
qp.mark(xx, yy)

qp.pen('777')
for n in range(len(xx)-1):
    qp.gline([[qp.AbsData(xx[n],yy[n]), qp.Retract(5)],
              [qp.AbsData(xx[n+1],yy[n+1]), qp.Retract(5)]])

qp.shrink()
qp.save('pdf')
